html-forms/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <label for="email">Email</label>
+    <div>
+      <input type="email">
+    </div>
+
+    <label for="options">Why is coding cool?</label>
+    <div>
+      <select>
+        <option>Select an option</option>
+        <option>Because it's interesting</option>
+        <option>Because it's fun</option>
+        <option>Because it's powerful</option>
+      </select>
+    </div>
+
+    <label for="languages">Favorite Language (choose one)</label>
+    <div>
+      <input type="radio" name="languages">
+      <label for="html">HTML</label>
+      <input type="radio" name="languages">
+      <label for="css">CSS</label>
+      <input type="radio" name="languages">
+      <label for="javascript">JavaScript</label>
+    </div>
+
+    <div>
+      <label for="newsletter">Would you like to sign up for more information?</label>
+      <div>
+        <input type="checkbox">
+        <label for="sign-up">Sign me up!</label>
+      </div>
+    </div>
+
+    <label for="comments">Comments</label>
```

Comment:
This `for` attribute here is correct, but the form control element it is paired with needs a matching `id` attribute with the same value.